diff --git a/node_modules/fastembed/lib/esm/fastembed.js b/node_modules/fastembed/lib/esm/fastembed.js
index 620362e..3e1ecd1 100644
--- a/node_modules/fastembed/lib/esm/fastembed.js
+++ b/node_modules/fastembed/lib/esm/fastembed.js
@@ -4,7 +4,7 @@ import https from "https";
 import * as ort from "onnxruntime-node";
 import path from "path";
 import Progress from "progress";
-import tar from "tar";
+import * as tar from "tar";
 import { downloadFileToCacheDir } from "@huggingface/hub";
 export var ExecutionProvider;
 (function (ExecutionProvider) {
